kitsune/questions/jinja2/questions/question_details.html

```diff
@@ -421,116 +421,118 @@ <h3 class="sumo-card-heading">{{ _('Post a Reply') }}</h3>
           {% endif %}
         </div>
       </li>
-      <li class="mzp-c-details">
-        <h3 class="sidebar-subheading sidebar-nav--heading-item is-accordion-heading">{{ _('Question Details') }}</h3>
-        <div id="question-details" class="question-details">
-          <form method="post" action="{{ url('questions.edit_details', question.id) }}">
-          <div class="folder">
-            {% if product or (user and user.has_perm('questions.change_question')) %}
-              <div class="sidebox tight condensed">
-                <span class="title">{{ _('Product') }}:</span>
-                {% if product %}
-                  <span class="detail">{{ pgettext('DB: products.Product.title', product.title) }}</span>
-                {% endif %}
-                {% if user and user.has_perm('questions.change_question') %}
-                  <select id="details-product" name="product">
-                    {% for p in all_products %}
-                      <option {% if product and p.id == product.id %}selected="selected"{% endif %} value="{{ p.id }}" data-url="{{ url('products.product', p.slug) }}">{{ p.title }}</option>
-                    {% endfor %}
-                  </select>
-                {% endif %}
-              </div>
-            {% endif %}
-
-            {% if topic or (user and user.has_perm('questions.change_question')) %}
-              <div class="sidebox tight condensed">
-                <span class="title">{{ _('Topic') }}:</span>
-                {% if topic %}
-                  <span class="detail">{{ pgettext('DB: products.Topic.title', topic.title) }}</span>
-                {% endif %}
-                {% if user and user.has_perm('questions.change_question') %}
-                  <select id="details-topic" name="topic">
-                    {% for t in all_topics %}
-                      <option {% if topic and t.id == topic.id %}selected="selected"{% endif %} value="{{ t.id }}">{{ t.title }}</option>
+      {% if is_trusted_user(request.user) %}
```

Comment:
This `if` statement is the only change in the template. Everything else is caused by the change in the indentation - 2 spaces in